### PR TITLE
Update versions, release notes, and changelog for 6.0.0 release

### PR DESCRIPTION
**Pull request by markcowl**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by markcowl** [comment]
signing run here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/512/

File: src/ResourceManager/Network/ChangeLog.md [hunk 1]
```diff
@@ -19,9 +19,8 @@
 --->
 
 ## Current Release
-* Bump up network sdk version from 17.0.0.preview to 18.0.0.preview
-* Rerecord all network tests with latest sdk version
-* Set minimum dependency of module to PowerShell 5.0
+## Version 6.0.0* Bump up network sdk version from 17.0.0.preview to 18.0.0.preview
```
Comment (maddieclayton) [inline src/ResourceManager/Network/ChangeLog.md]: Move onto new line

**Comment by markcowl** [comment]
signing: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/514/

**Comment by markcowl** [comment]
signing: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/519/
on demand: https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/794/

**Pull request merged**